vars/packageBuildingPipelineDAOS.groovy

```diff
@@ -139,6 +139,56 @@ def call(Map pipeline_args) {
             } //stage('Lint')
             stage('Build') {
                 parallel {
+                    stage('Coverity') {
+                        when {
+                            beforeAgent true
+                            allOf {
+                                expression { pipeline_args.get('coverity','') != '' }
```

Comment:
This would match on `false` wouldn't it?  Maybe we should be looking for `true` explicitly so that one can toggle it on and off with `true` and `false` and not have to delete the option altogether?

Reply:
The parameter needs to be a Coverity project name, true and false are both invalid,  So either no parameter, or a parameter of '' means no Coverity build.

Reply:
Ah, yes, I see that below.